edisgo/flex_opt/heat_pump_operation.py

```diff
@@ -31,11 +32,37 @@ def operating_strategy(
     if heat_pump_names is None:
         heat_pump_names = edisgo_obj.heat_pump.cop_df.columns
 
+    missing = set(heat_pump_names) - set(edisgo_obj.topology.loads_df.index)
+    if missing:
+        logger.warning(
+            f"The following heat pumps are are in the heat pump class but not yet "
+            f"integrated into the topology class. Therefore, their maximum capacity "
+            f"cannot be considered in the operating strategies. {missing=}"
```

Comment:
Add a note similar to "For better results, please integrate these heat pumps into the topology before running this function.".